src/Simulation/Simulators/OpenSystemsSimulator/Decompositions/README.md

```diff
@@ -0,0 +1,4 @@
+# TODO
+
+The decompositions in this folder should eventually be converted into a new "type 4" targeting package.
+We use the legacy approach to making intrinsics available to simulators, however, on a temporary basis for compatibility with Python and IQ#.
```

Comment:
Makes sense, and hopefully we can finish up the support soon such that this workaround won't be needed anymore!

Reply:
Sounds good! Do we have an issue for Python/IQ# ↔ QIR integration that I can link to here?

Reply:
We don't, actually... let me connect with @bettinaheim and get the right thing filed in the right place for tracking.